web/migrations/0025_educationalvideo.py

```
@@ -0,0 +1,62 @@
+# Generated by Django 5.1.6 on 2025-03-13 16:18
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+from django.utils.text import slugify
```

Ruff (0.8.2)

`django.utils.text.slugify` imported but unused

Remove unused import: `django.utils.text.slugify`

(F401)